lib/ecom/api.ts

```diff
@@ -103,6 +86,29 @@ function createApi(): EcomAPI {
                 return failureResponse(EcomApiErrorCodes.GetProductsFailure, getErrorMessage(e));
             }
         },
+        async getFeaturedProducts(categorySlug, count) {
```

Comment:
I don't think it should be a part of the API

Reply:
Why not?

Reply:
I don't see any crusial reasons why this should or shouldn't be a part of an API. so, @alisey asked to move it to API, and @yurii-ve said this shouldn't be there. lets vote? 😀

Reply:
My reasoning is that (if I'm not missing anything) API doesn't have notion of 'Featured Products', it's purely ui thing, and I don't like the idea of mixing API specific functionality with client code that just happen to call API.
But given that we already have `getPromotedProducts` below, maybe it's ok

Reply:
`getPromotedProducts` needs to be removed. It's unused, and conceptually it's the same thing as featured products.

`EcomAPI` is our own interface. It wraps `WixClient` but doesn't map one to one, even for methods like `getOrder`.

`WixClient` lacks useful methods like `getProductPriceBounds`, so we've had to add our own. Same goes for `getFeaturedProducts`, it's basically sort by recommended, with a fallback to all products.